backend/src/bundles/users/user.controller.ts

```diff
@@ -24,6 +26,7 @@ import { UsersApiPath } from './enums/enums.js';
  *            type: string
```

Comment:
update user entity

Reply:
fix in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/98/commits/5b2b613ffbe6869c2f7f6186207b4c9bf8f25abe](url)